src/actors/FilterBarActor.js

```diff
@@ -0,0 +1,88 @@
+import * as SharedUtils from '../utils/SharedUtils';
+
+export class FilterBarActor {
```

Comment:
@jasonl Using this implementation is trying to follow the one way data flow without added boiler plate of the dispatcher. Having worked on a large backbonejs application before I have an opinion about using events for everything. Things I've noticed
1. It works
2. It becomes incredibly tricky to know exactly what will happen when an event is triggered. Eg. If I click bottom 'a' I want it to change model 'b'. Sounds simple but what if changing model b also updates model 'c' which updates model... all the while the views are listening for these changes and updating as they go. It gets slow and complicated very quickly.

By using these "Actors", which are just a collection of methods that have access to some common dependencies there is only 1 path through the event which greatly simplifies the intent of the code and also reduces extra view re-renders.